.github/workflows/kubernetes-deploy.yaml

```diff
@@ -72,17 +78,12 @@ jobs:
           echo $GATEWAY_HOST
           # basic tests
           cd /home/runner/work/quantum-serverless/quantum-serverless/tests/basic
-          for f in *.py; do echo "$f" && python "$f"; done
-          # experimental tests
-          cd /home/runner/work/quantum-serverless/quantum-serverless/tests/experimental
-          for f in *.py; do echo "$f" && python "$f"; done
+          for f in *.py; do echo "$f" && python "$f" && kubectl get raycluster -o yaml; done
```

Comment:
What does this piece of code do exactly, @akihikokuroda ?

Reply:
I'm debugging this test failure.  I'll take it out when it's done.  
Thanks!